src/vs/workbench/contrib/webview/browser/webviewElement.ts

```diff
@@ -88,6 +88,14 @@ export class WebviewElement extends Disposable implements IWebview, WebviewFindD
 	 */
 	public readonly origin: string;
 
+	private readonly _codeWindow: CodeWindow;
+
+	/**
+	 * The code window the webview is contained within.
+	 */
+	public get codeWindow(): CodeWindow { return this._codeWindow; }
```

Comment:
Do we need this? Storing the full `window` object on a class makes me a bit nervous because there is a risk of leakage. I often end up just keeping the `windowId` instead.

Reply:
This can be removed from the typings but yeah, internally it should also just store the windowId.